qi-doc/scribblings/qi.scrbl

```diff
@@ -40,6 +40,7 @@ This site hosts @emph{user} documentation. If you are interested in contributing
 @include-section["tutorial.scrbl"]
 @include-section["interface.scrbl"]
 @include-section["forms.scrbl"]
+@include-section["list-operations.scrbl"]
```

Comment:
The subject of 53532d9e0daef934c8a368faae7c0e18a7d84830 needs work. Perhaps "add qi/list bindings" with more detail in the
body (don't underestimate a good commit message body)?

Reply:
Addressed in the rebases.